src/open_clip/openai.py

```diff
@@ -44,7 +47,7 @@ def load_openai_model(
         A torchvision transform that converts a PIL image into a tensor that the returned model can take as its input
     """
     if get_pretrained_url(name, 'openai'):
-        model_path = download_pretrained_from_url(get_pretrained_url(name, 'openai'))
+        model_path = download_pretrained_from_url(get_pretrained_url(name, 'openai'), cache_dir)
```

Comment:
use kw arg here for cache_dir for future possible refactoring... (less chance of break)